Integrations/integration-ServiceNow.yml

```diff
@@ -3,7 +3,6 @@ commonfields:
   version: -1
 name: ServiceNow
 display: ServiceNow
-fromversion: 2.5.0
```

Comment:
I don't think you intended to remove this.
Didn't we have SN prior to 2.5?
if so, we should add another integration with to version 2.5, before those changes